src/EditorFeatures/VisualBasicTest/Formatting/Indentation/SmartIndenterTests.vb

```diff
@@ -2492,7 +2482,7 @@ End Namespace
             AssertSmartIndent(
                 code,
                 indentationLine:=3,
-                expectedIndentation:=Nothing,
+                expectedIndentation:=0,
```

Comment:
this is an improvement.  previously, the indentation service couldn't answer these basic questions.  so features that needed the info were simply SOL.

Reply:
you probably need to test this scenario. we might be trying to override existing smart indent behavior and if we return someting, editor doesn't give us control and they do it themselves breaking our existing behavior on enter.

with so many behavior changes, wonder whether this is okay.

also, if we 100% sure this is right thing to do, we should remove all the case where ISmartIndent return null - basically remove all those legacy code.

Reply:
> we should remove all the case where ISmartIndent return null - basically remove all those legacy code.

That's what my PRs get us to.

Reply:
I think  you are confusing with editor's ISmartIndenter which is for editor and generic indentation service that you want to use in code fix.

I am fine changing indentation service's behavior, but not sure why you want to chagne ISmartIdenter's behavior.